src/factors/Range2D.jl

```diff
@@ -13,17 +13,13 @@ Point2Point2Range(d::D) where {D <: IIF.SamplableBelief} = Point2Point2Range{D}(
 function getSample(cfo::CalcFactor{<:Point2Point2Range}, N::Int=1)
   return (reshape(rand(cfo.factor.Z,N),1,N),  2*pi*rand(N))
 end
-function (cfo::CalcFactor{<:Point2Point2Range})(
-            res::AbstractVector{<:Real},
-            rho,
-            theta,
-            xi,
-            lm )
+function (cfo::CalcFactor{<:Point2Point2Range})(rho, theta, xi, lm)
   #
   XX = lm[1] - (rho[1]*cos(theta[1]) + xi[1])
   YY = lm[2] - (rho[1]*sin(theta[1]) + xi[2])
-  res[1] = XX^2 + YY^2
-  return nothing
+  #TODO JT - Should this have a sqrt for parametric?
```

Comment:
my feeling is yes, else the Malahanobis distance will be wrong.  else factors will be mixing ^2 and ^4 costs.  sort is a little excess but safest path for now i think

Reply:
See this comparison from slack: "Actually the getSample theta part (2*pi*rand(N)) won't work in any case"
https://github.com/JuliaRobotics/RoME.jl/compare/feat/21Q1/factor_residuals...twig/21Q1/parametricPoint2Point2Range?expand=1